mmv1/third_party/terraform/utils/header_transport.go

```diff
@@ -29,6 +30,11 @@ func (h headerTransportLayer) RoundTrip(req *http.Request) (*http.Response, erro
 		// only set headers that are not previously defined
 		if _, ok := req.Header[key]; !ok {
 			req.Header[key] = value
+		} else if key == "User-Agent" {
```

Comment:
This should be obsoleted by just modifying Config.userAgent.

Reply:
done